### DISCUSSION TITLE
Shouldn't `ifconfig down` disable the radio?

### DISCUSSION THREAD

**Discussion by edmont** [openthread/openthread#9143] [Q&A] [answered]
I would expect that `otIp6SetEnabled(false)` effectively disables the radio, but this doesn't seem to be the case. Actually it seems `Links::Disable` is never used in the code. Is there any reason for that?

**Comment by abtink** [comment]
I cannot recall any specific reason for this. 
I noticed this (that we never call `Disable()`) in past. To me it is reasonable to add this but may be careful if there happens to be any side-effects

**Comment by jwhui** [comment, marked as answer]
I believe the historical reason is that there may be other users of the 15.4 radio. For example, raw 15.4 radio usage, which may include proprietary protocols leveraging the raw link API.

OpenThread does provide `otLinkSetEnabled()` for controlling the radio state.

**Comment by edmont** [comment]
Thanks for your replies. That's a good point @jwhui, however even `otLinkSetEnabled(false)` is not turning the radio off today. I created https://github.com/openthread/openthread/pull/9156 to address the issue.
